packages/wallets/src/features/cfd/screens/ResubmitPOA/ResubmitPOA.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useSettings, useStatesList } from '@deriv/api';
+import { InlineMessage, WalletDropdown, WalletText, WalletTextField } from '../../../../components/Base';
+import './ResubmitPOA.scss';
+
+const ResubmitPOA: React.FC = () => {
+    const { data } = useSettings();
+    const country = data?.country_code || '';
+    const { data: statesList } = useStatesList(country);
+
+    const [selectedState, setSelectedState] = useState('');
+
+    const handleSelect = (value: string) => {
+        setSelectedState(value);
+    };
+
+    return (
+        <div className='wallets-resubmit-poa'>
+            <div className='wallets-resubmit-poa__address'>
+                <div className='wallets-resubmit-poa__address__label'>
+                    <WalletText weight='bold'>Address</WalletText>
+                    <div className='wallets-resubmit-poa__address__divider' />
```

Comment:
Can we use Divider component

Reply:
Divider component is not apt for this use case